### PR TITLE
Slightly improves cookiecutter accessibility.

### PR DESCRIPTION
**Pull request by zooba**

### CODE REVIEW

File: Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ class CategorizedViewModel : INotifyPropertyChanged {
             _displayName = displayName;
         }
 
-        public bool Selectable => false;
+        public bool Selectable => true;
```
Comment (huguesv) [inline Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs]: If you haven't already, check that things work properly when the category is selected. For example, menu/toolbar items disabled as appropriate, template description section at the bottom is correct.

File: Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml [hunk 2]
```diff
@@ -146,7 +145,8 @@
                 <HierarchicalDataTemplate DataType="{x:Type vm:CategorizedViewModel}" ItemsSource="{Binding Path=Templates}">
                     <TextBlock Margin="0 4 0 4"
                                FontWeight="Bold"
-                               Text="{Binding Path=DisplayName}"/>
+                               AutomationProperties.ItemType="Category"
```
Comment (huguesv) [inline Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml]: Looks like "Category" should be localized?  It gives examples of "Document File" and "Folder".
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationproperties.itemtype(v=vs.110).aspx

**Comment by huguesv** [comment]
Looks good otherwise

File: Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml (hunk 2 above)
Comment (zooba) [inline Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml]: It doesn't work anyway - I'll remove it.

File: Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs (hunk 1 above)
Comment (zooba) [inline Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs]: Thought I checked this, but I'll check again.

Comment (zooba) [inline Python/Product/Cookiecutter/ViewModel/CategorizedViewModel.cs]: Yep, it works fine.

**Pull request merged**